lib/Settings/About.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:url_launcher/url_launcher.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+
+class About extends StatelessWidget {
+  final controller = ScrollController();
```

Comment:
This looks like it's not used...? What does this do?

Reply:
For the images in `assets/help` they are not icons but mostly instructions for using Cboard that should be included in the `User-help` markdown
Regrading your second comment about `lib/Settings/` and `translation/About`,  I'm not sure I understand it correctly. Could you explain again ? 😀

Reply:
> This looks like it's not used...? What does this do?

It was my mistake. I first tried it for TOC but forgot to cross it out.